roles/rsyslog/README.md

```diff
@@ -5,22 +5,6 @@ linux-system-roles-rsyslog
 
 This is the ansible rsyslog roles to deploy configuration files.
 
-Table of Contents
-=================
-
-<!--ts-->
-   * [linux-system-roles-rsyslog](#linux-system-roles-rsyslog)
-   * [Table of contents](#table-of-contents)
-   * [Deploy Rsyslog Configuration Files](#deploy)
-      * [Inventory File](#inventory-file)
-      * [vars.yml](#vars)
-      * [playbook.yml](#playbook)
-   * [Variables in vars.yml](#variables)
-      * [Common sub-variables](#common-variables)
-      * [Viaq sub-variables](#viaw)
-   * [Contents of Roles](#roles)
-<!--te-->
-
```

Comment:
Was it intended to drop the table of contents?

Reply:
Yeah, it the table of contents are obsolete... I think I should update the READMEs more readable and useful for the endusers...